src/sample/classes/StudentList.java

```diff
@@ -0,0 +1,66 @@
+package sample.classes;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+public class StudentList {
+    
+    String name;
+    int matric;
+
+    public StudentList(){
+        this.name = "";
+        this.matric = 0;
+    }
+    
+    
+    public String getname() {
+        return name;
+    }
+
+    public void setname(String n) {
+        this.name = n;
+    }
+
+    public int getmatric() {
+        return matric;
+    }
+
+    public void setmatric(int m) {
+        this.matric = m;
+    }
+
+    public StudentList(String name, int matric){
+        this.name = name;
+        this.matric = matric;
+    }
+    
+    /*
+    StringProperty name = new SimpleStringProperty();
```

Comment:
If the comments are not necessary, please remove it ya

Reply:
The comments are actually the class needed for the Student Name List according to https://stackoverflow.com/questions/34424030/take-a-txt-file-from-computer-and-insert-it-tableview-javafx I make this part as comment because I am using the class from this one https://stackoverflow.com/questions/65037615/how-to-import-csv-or-txt-files-to-be-displayed-on-tableview-of-high-score-menu but both approaches cannot display the result on the TableView too